content/news/009.cc-in-prod.en.md

```diff
@@ -0,0 +1,13 @@
+---
+published: 2024-07-11
+description: DD-IX now present in Dresdens digital heart.
+title: City Center PoP now in Operation
+keywords: [ Tech, Update ]
+authors: [ Tassilo ]
+image: 009_cc_pop.webp
+---
+
+This afternoon our [City Center (CC)](https://www.peeringdb.com/fac/15106) PoP went into production. This is an 
```

Comment:
This afternoon, we connected our PoP at ...

Reply:
This afternoon we put our [City Center (CC)](https://www.peeringdb.com/fac/15106) into operation.

Reply:
This afternoon we put our PoP at City Center (CC) into production.